### PR TITLE
Small Fix for year to display after resuming draft

### PR DESCRIPTION
**Pull request by orelbn**
*Issue #:* /bcgov/entity#16443

*Description of changes:*
* Removed parsing to string, and converting to number when setting as it is not needed for the select.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrRegistration/YourHome/ManufacturedYearSelect.vue [hunk 1]
```diff
@@ -75,13 +75,11 @@ export default defineComponent({
...
       currentYear: new Date().getFullYear(),
-      yearOfManufacture: getMhrRegistrationYearOfManufacture.value?.toString()
+      yearOfManufacture: getMhrRegistrationYearOfManufacture?.value
     })
 
-    watch(() => localState.yearOfManufacture, (val: string) => {
-      if (parseInt(val)) {
-        setMhrHomeBaseInformation({ key: 'year', value: parseInt(val) })
-      }
+    watch(() => localState.yearOfManufacture, (val: number) => {
```
Comment (dimak1) [inline ppr-ui/src/components/mhrRegistration/YourHome/ManufacturedYearSelect.vue:81]: Was the Year field changed to numbers only?

Comment (orelbn) [reply ppr-ui/src/components/mhrRegistration/YourHome/ManufacturedYearSelect.vue:81]: For the select yes: 

image

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1417-xy9p6z9r.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1417-xy9p6z9r.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1417-xy9p6z9r.web.app

**Review by cameron-eyds — APPROVED** [review]
Looks good!

File: ppr-ui/src/components/mhrRegistration/YourHome/ManufacturedYearSelect.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/YourHome/ManufacturedYearSelect.vue:81]: I didn't realize this was a pre-defined select, thanks!

**Review by dimak1 — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1417-xy9p6z9r.web.app

**Pull request merged**